test/visualisation/latexify.jl

```diff
@@ -247,18 +258,8 @@ let
     extended = extend(decaying_rate, base_network)
 
     # Latexify.@generate_test latexify(extended)
-    @test_broken latexify(extended) == replace(
-    raw"\begin{align*}
-    \mathrm{X} &\xrightarrow{k r} \varnothing  
-    0 &= -1 - x\left( t \right)  
-    \end{align*}
-    ", "\r\n"=>"\n")
```

Comment:
While we could generate tests they would point towards LaTeX code we wouldn't want, so I removed it and just marked as something broken.